README.md

```diff
@@ -8,13 +8,16 @@ This document contains information on how to get setup with the Simulation frame
 
 For more detailed information on the inner workings of the simulation please refer to the [wiki](https://github.com/UBC-Solar/Simulation/wiki).
 
+For instructions on how to use the scripts provided, see our [usage guide](simulation/cmd/USAGE_GUIDE.md).
+
 ## Getting started
 
 ### Prerequisites
 
-- Python 3.8 or above (https://www.python.org/downloads/)
+- Python 3.9 or above (https://www.python.org/downloads/)
```

Comment:
![image](https://github.com/user-attachments/assets/1df770c7-b3ea-4bcd-ada5-24e8d05ac1ca)

Python 3.9 does not seem to be supported for me

Reply:
worked for me (Python 3.9.13) maybe it's a local issue? @lambpasta

Reply:
I have fixed the README to state Python 3.10 is required